SPEC.md

```diff
@@ -86,6 +86,7 @@
   * [File write_objects(Array\[Object\])](#file-write_objectsarrayobject)
   * [File write_json(mixed)](#file-write_jsonmixed)
   * [File size(File, \[String\])](#float-sizefile-string)
+  * [File sub(String, String, String)](#string-substring-string-string)
```

Comment:
I don't really advertise this much, but I wrote a tool called `mdtoc` which is useful for stuff like this

```
$ pip install mdtoc
$ mdtoc SPEC.md
```

Seems that the `size()` function link might be wrong too?